src/Analyzers/Core/Analyzers/FileHeaders/FileHeader.cs

```diff
@@ -16,6 +16,11 @@ namespace Microsoft.CodeAnalysis.FileHeaders
         /// </summary>
         private readonly int _fileHeaderStart;
 
+        /// <summary>
+        /// The location in the source where the file header was expected to end.
+        /// </summary>
+        private readonly int _fileHeaderEnd;
```

Comment:
would prefer we just make this `_fileHeaderSpan`.

Reply:
Done in 099dcdd4